src/Pages/ProductsList/Item.js

```diff
@@ -0,0 +1,113 @@
+import React, { useRef, useState, useEffect } from "react";
+import styled from "styled-components";
+import New from "./New";
+import Best from "./Best";
+
+const useHover = () => {
+  const [value, setValue] = useState(false);
+
+  const ref = useRef(null);
+
+  const handleMouseOver = () => setValue(true);
+  const handleMouseOut = () => setValue(false);
+
+  useEffect(() => {
+    const node = ref.current;
+    if (node) {
+      node.addEventListener("mouseover", handleMouseOver);
+      node.addEventListener("mouseout", handleMouseOut);
+
+      return () => {
+        node.removeEventListener("mouseover", handleMouseOver);
+        node.removeEventListener("mouseout", handleMouseOut);
+      };
+    }
+  }, []);
+  return [ref, value];
+};
+
+function Item({
+  isNew,
+  isBest,
+  image,
+  hashFirst,
+  hashSecond,
+  productName,
+  hover,
+}) {
+  const [hoverRef, isHoverd] = useHover();
+
+  return (
+    <ItemOuter>
+      <ItemImg ref={hoverRef} src={isHoverd ? hover : image} />
+      <Mark>
+        {isNew === "true" ? <New /> : null}
```

Comment:
```suggestion
        {isNew === "true" && <New />}
```
true만 렌더할 경우 &&를 써주세요

Reply:
적용시켰습니다.₩~~~~